base/dsfs/structure.go

```diff
@@ -8,6 +8,15 @@ import (
 	"github.com/qri-io/qfs/cafs"
 )
 
+// BaseTabularSchema is the base schema for tabular data
```

Comment:
When we start supporting object-of-equal-length-array tables as a format, this'll explode..

We should add a note here that this shouldn't be used. Instead favor methods in `github.com/qri-io/dataset/tabular`. 

The even better thing to do would be to move this into the `tabular` package (not in this PR, some other time) and rename it to `BaseSchemaArray`, like we do with JSON identity schemas

Reply:
I used to be able to use dataset's BaseSchemaArray, but some tests were failing that "items" needed to exist at the top-level. I think some lower codepath has started relying on arrays always being 2-dimensional?